src/styles/blocks/review.scss

```diff
@@ -0,0 +1,30 @@
+.review {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
```

Comment:
Ensure that the `font-size` of 10px adheres to the design specifications. If the design specifies a different `font-size`, update it accordingly.